integreat_cms/cms/views/pages/page_form_view.py

```diff
@@ -176,6 +177,7 @@ def get(self, request, *args, **kwargs):
             },
         )
 
+    @transaction.atomic
```

Comment:
Maybe let's add this to the post-function of `LanguageTreeNodeFormView` as well?

Reply:
Yes, I missed that one.